.stylelintrc.json

```diff
@@ -1,6 +1,16 @@
 {
   "extends": ["stylelint-config-standard", "stylelint-config-standard-scss"], 
   "rules": {
-    "value-list-comma-newline-after": "always-multi-line"
+    "value-list-comma-newline-after": "always-multi-line",
+    "function-no-unknown": [
+      true,
+      { "ignoreFunctions": [
```

Comment:
Just the tiniest white-space nitpick: can we move `ignoreFunctions` down to the next line so the curly brace is all by itself?